podpac/core/coordinates/coordinates.py

```diff
@@ -830,13 +865,61 @@ def intersect(self, other, outer=False, return_indices=False):
             List of indices for each dimension that produces the intersection, only if ``return_indices`` is True.
         """
 
-        intersections = [c.intersect(other, outer=outer, return_indices=return_indices) for c in self.values()]
+        intersections = [c.intersect(other, outer=outer, return_indices=return_indices) for c in self._coords.values()]
+        return self._make_selected_coordinates(intersections, return_indices)
+
+    def select(self, lat=None, lon=None, alt=None, time=None, return_indices=False, outer=False):
+        """
+        Get the coordinate values that are within the given bounds for each dimension.
+
+        The default selection returns coordinates that are within the bounds::
+
+            In [1]: c = Coordinates([[0, 1, 2, 3], [10, 20, 30, 40], dims=['lat', 'lon'])
+
+            In [2]: c.select(lat=[1.5, 2.5])
+            Out[2]: TODO

```

Comment:
want to fill in the TODO items for select - these have errors, see next comment

Reply:
I also see a different error when running your PR in the notebook:

It looks like we would want this to be **dict:

```
c.select({'lat': [1, 3]})

>>

KeyError                                  Traceback (most recent call last)
<ipython-input-71-d7c9df0f2790> in <module>()
----> 1 c.select({'lat': [1, 3]})

~/computing/creare/podpac/podpac/podpac/core/coordinates/coordinates.py in select(self, lat, lon, alt, time, return_indices, outer)
    909 
    910         bounds = {'lat':lat, 'lon':lon, 'alt':alt, 'time':time}
--> 911         selections = [c.select(bounds, outer=outer, return_indices=return_indices) for c in self._coords.values()]
    912         return _make_selected_coordinates(selections, return_indices)
    913 

~/computing/creare/podpac/podpac/podpac/core/coordinates/coordinates.py in <listcomp>(.0)
    909 
    910         bounds = {'lat':lat, 'lon':lon, 'alt':alt, 'time':time}
--> 911         selections = [c.select(bounds, outer=outer, return_indices=return_indices) for c in self._coords.values()]
    912         return _make_selected_coordinates(selections, return_indices)
    913 

~/computing/creare/podpac/podpac/podpac/core/coordinates/coordinates1d.py in select(self, bounds, return_indices, outer)
    478                 return self._select_full(return_indices)
    479 
--> 480         bounds = make_coord_value(bounds[0]), make_coord_value(bounds[1])
    481 
    482         # full

KeyError: 0
```